v4/weaviate/graphql/multiclassbuilder.go

```diff
@@ -31,7 +32,13 @@ func (mb *MultiClassBuilder) Do(ctx context.Context) (*models.GraphQLResponse, e
 // build the GraphQL query string (not needed when Do is executed)
 func (mb *MultiClassBuilder) build() string {
 	var query string
-	for _, class := range mb.classBuilders {
+	// sorting classBuilder based on className to have consistent order in query
+	s := make(classBuildersSorter, 0, len(mb.classBuilders))
+	for _, k := range mb.classBuilders {
+		s = append(s, k)
+	}
+	sort.Sort(classBuildersSorter(s))
```

Comment:
Instead of sorting  the classes, could we just add the map keys (eg the classnames to a slice) and sort those and then iterate over those afterwards? Then we do not need to add the sort interface.

Or another idea:
If we need a slice anyway, how about adding the keys directly to it? Then we would preserve the original order in which users add classes, which might be helpful?

Reply:
Makes sense to do that. I've also fixed the failing tests.

Reply:
If Risotto is deleted again that should also fix those node tests that now error because of an additional class

Reply:
Sorry I didn't saw your comment before making `node test fix` commit. So the issue is, in most of the tests we are using `CreateTestSchemaAndData` function to setup classes and add data. Now, I didn't realize this before and because of that in each test i.e. node or backup we are creating one extra class `Risotto`. I thing it would be better to create a separate function just for `MultiClassBuilder` tests to setup and add data, which can insert `Risotto` class as well.

Reply:
Yes there is no dependency. We can go ahead and merge this.

Reply:
great, merging then. Thanks for your contribution!

I added two tickets with the [good first issue](https://github.com/weaviate/weaviate-go-client/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) label in case you want to tackle something else.